dts/arm/st/l4/stm32l4.dtsi

```diff
@@ -478,6 +478,15 @@
 			status = "disabled";
 		};
 
+		dcmi: dcmi@50050000 {
+			compatible = "st,stm32-dcmi";
+			reg = <0x50050000 0x400>;
+			interrupts = <85 0>;
+			interrupt-names = "dcmi";
+			clocks = <&rcc STM32_CLOCK_BUS_AHB2 0x00004000>;
+			status = "disabled";
+		};
```

Comment:
This node is not present in all STM32L4 series so, it should be added only in impacted series dtsi files: stm32l496.dtsi and stm32l4p5.dtsi (which will then benefit to all .dtsi including those).

Reply:
Will do! Where did you see this?

Reply:
I've looked for definitions in following files: https://github.com/zephyrproject-rtos/hal_stm32/tree/main/stm32cube/stm32l4xx/soc

Reply:
I've pushed the changes.
Just so I understand since I'm pretty new to zephyr - I looked at the files you pointed to but I do see DCMI definitions in other files such as stm32l4s5xx.h.